rules/autoconsent/flex-orange-pl.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "flex-orange-pl",
+    "cosmetic": true,
+    "prehideSelectors": ["div[uib-section=\"home\"] > div > div:nth-child(4) > div"],
+    "detectCmp": [{ "exists": "div[uib-section=\"home\"] > div > div:nth-child(4) > div" }],
```

Comment:
This didn't trigger for me, maybe the dom structure has already changed. Looking at the markup, this seems like a difficult one to support, I'd suggest dropping it, unless you think it's easy to fix.

Reply:
The structure did change. `div:nth-child(4)` <- This bit went up to one level (3 instead of 4). It does sound unstable so yes I agree to drop it.